application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCounter.java

```diff
@@ -0,0 +1,39 @@
+package org.togetherjava.tjbot.features.basic;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Guild;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.features.Routine;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class MemberCounter implements Routine {
+    private final String wordsInCategory;
+
+    public MemberCounter(Config config) {
+        this.wordsInCategory = config.getMemberCountCategoryName();
+    }
+
+    @Override
+    public Schedule createSchedule() {
+        return new Schedule(ScheduleMode.FIXED_RATE, 0, 12, TimeUnit.HOURS);
+    }
+
+    @Override
+    public void runRoutine(JDA jda) {
```

Comment:
Maybe we can improve the readability of this method. Let's split it up for example seperating the logic:

 ```java
private void updateCategoryName(Category category) {
    int totalMemberCount = category.getGuild().getMemberCount();
    String basename = category.getName();
    category.getManager()
        .setName("%s - %d Members".formatted(basename, totalMemberCount))
        .queue();
}
```